testgrid/config.yaml

```diff
@@ -4594,56 +4594,76 @@ dashboards:
   dashboard_tab:
   - name: build-master
     test_group_name: ci-kubernetes-build
+    description: TODO: think this is owned by sig-release release-engineering subproject, set an alert up; TODO: what file does this come from; TODO: what does this do
   - name: integration-master
     test_group_name: ci-kubernetes-integration-master
+    description: TODO: think this is owned by sig-release release-team subproject, set an alert up; TODO: what file does this come from; TODO: what does this do
   - name: bazel-build-master
     test_group_name: ci-kubernetes-bazel-build
+    description: TODO: think this is owned by sig-release release-engineering subproject, set an alert up; TODO: what file does this come from; TODO: what does this do
   - name: bazel-test-master
     test_group_name: ci-kubernetes-bazel-test
+    description: TODO: think this is owned by sig-testing, set an alert up; TODO: what file does this come from; TODO: what does this do
   - name: verify-master
     test_group_name: ci-kubernetes-verify-master
+    description: TODO: think this is owned by sig-testing, set an alert up; TODO: what file does this come from; TODO: what does this do
```

Comment:
in this case I feel like sig-testing has the folks who most quickly know what's going wrong, so I strongly suspect failures would get routed there anyway, even if it looks similar to an umbrella/test-routing job akin to unit/integration/e2e

Reply:
verify and to a lesser extend integration are total kitchen sink jobs, again the people that actually tend to handle this are pretty present in both 🤷‍♂️

Reply:
yeah... sig-testing probably can own these verify bashes, integration is mostly different sigs...